code/modules/mob/living/living.dm

```diff
@@ -1024,3 +995,7 @@ Sorry Giacom. Please don't be mad :(
 		for(var/datum/objective/sintouched/acedia/A in src.mind.objectives)
 			return 1
 	return 0
+
+/mob/living/throw_at(atom/target, range, speed, mob/thrower, spin=1, diagonals_first = 0)
+	stop_pulling()
```

Comment:
By adding these stop_pulling in throw_at(), we no longer need to be checking for "throwing" in living/Move() when pulling something.